packages/plugin-lambda/src/commands/instrument.ts

```diff
@@ -319,30 +320,34 @@ export class PluginCommand extends LambdaInstrumentCommand {
   }
 
   private getSettings(): InstrumentationSettings | undefined {
-    const layerVersionStr = this.layerVersion ?? this.config.layerVersion
-    const extensionVersionStr = this.extensionVersion ?? this.config.extensionVersion
+    const layerVersionStr = this.layerVersion ?? this.config.layerVersion ?? 'none'
```

Comment:
Ah I see, if not set, you just set it to `none`, so essentially, following this `null` pattern,

I'm just confused, can't we just expect it to be `null` instead of using an explicit `none` string?

What's the advantage of this?